client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	// Automattic widget ID
+	id: '8a2968fc57d1e2f40157f42bf2d43160',
+	// By default don't show the "Ask a Question" button
+	displayAskQuestion: false
+};
+
+let initialized = false;
+
+const loadDirectlyScript = ( callback = noop ) => {
```

Comment:
Sure. There's a couple SNAFUs with immediately executing the vendor script like that (if you recently asked a question the widget will pop up regardless of what page you're on; if the config hasn't been set yet, the lib falls over). So I'd likely need to wrap the code in a function and export that, so it's only executed once config has been set and you're ready for the widget to appear.

I'm also curious how this affects the larger conversation happening now about performance in Calypso? Granted that script is pretty tiny (~5KB gzip'd) but bundling it in still means sending this code across to a majority of users who will never need it. Should we worry about that now, or is that something we assume will be handled later by code-splitting?

And I suppose my other concern is about updates from the vendor — a nicety of linking to the remote script is we can do nothing and receive changes (bugfixes, new features, etc) from Directly. Locking in like this means at best we're frozen to a version of the widget, but at worst the widget could silently breaks on us (`embed.js` pulls in a host of other dependencies which _could_ change under us, expecting `embed.js` to have also changed in some critical way). My instinct is that the safest thing is to model our implementation in the spirit of the vendor's instructions as much as possible, but if that's not standard procedures here that's 👌 with me.

(Sorry for so many questions, but I'm trying to make sure I understand the Calypso way of doing things :smile:)